definitions/npm/apollo-link-http_v1.2.x/flow_v0.28.x-/apollo-link-http_v1.2.x.js

```diff
@@ -0,0 +1,20 @@
+declare module "apollo-link-http" {
+  declare export interface FetchOptions {
+    uri?: string;
+    //todo change to type of Global['fetch']
+    fetch?: any;
+    includeExtensions?: boolean;
+    credentials?: string;
+    headers?: any;
+    fetchOptions?: any;
+  }
+
+  //todo change return type to ApolloLink from 'apollo-link'
+  declare export function createHttpLink(opts: FetchOptions): any;
+
+  declare export class HttpLink {
+    //todo change type to RequestHandler from 'apollo-link'
+    requester: any;
```

Comment:
The type of `requester` is `RequestHandler` from the `apollo-link` package. What is the appropriate way to declare this? (Keep in mind, `apollo-link` might not have lib defs either.)

Reply:
Dependencies between libdefs will probably been handled in the coming `3.0` release, which is currently discussed in #1494.

For now I think it's best to just duplicate the `RequestHandler` inside this libdef (like it's done in `react-router-dom` & `react-router-native` for the duplicate types from `react-router`) and add a `// TODO:` to delete this in the future when dependencies between libdefs are handled.